stable/elasticsearch/templates/configmap.yaml

```diff
@@ -18,6 +18,8 @@ data:
     node.ingest: ${NODE_INGEST:true}
 {{- else if hasPrefix "6." .Values.appVersion }}
     node.ingest: ${NODE_INGEST:true}
+{{- else if hasPrefix "7." .Values.appVersion }}
```

Comment:
Could all those lines be factored?

Reply:
Do you mean using semver compare?

Reply:
Hm, just factoring all those lines only setting node.ingest, like `if or (hasPrefix X .Values.appVersion) (hasPrefix Y .Values.appVersion)`